### PR TITLE
Feature: Abebe's Besso Eating Ability

### PR DESCRIPTION
**Pull request by Bela-Jash**

### CODE REVIEW

File: Abebe.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Abebe besso bela.
```
Comment (Abel-Tessema) [inline Abebe.txt:1]: Make this line more descriptive.

**Review by Abel-Tessema — CHANGES_REQUESTED** [review]
Make these changes, and push again.

File: Abebe.txt (hunk 1 above)
Comment (Bela-Jash) [reply Abebe.txt:1]: Alright, I'm on it.

**Review by Abel-Tessema — APPROVED** [review]
Everything looks great now. Merge ahead!

**Pull request merged**